closed/OpenJ9.gmk

```diff
@@ -179,6 +179,14 @@ $(foreach subdir, j9vm server, \
 		$(subdir)/$(call SHARED_LIBRARY,jvm), \
 		redirector/$(call SHARED_LIBRARY,jvm_jdk11)))
 
+# jsig
+$(call openj9_add_jdk_special, \
+                server/$(call SHARED_LIBRARY,jsig), \
+                $(call SHARED_LIBRARY,jsig))
+$(call openj9_add_jdk_special, \
+                $(call SHARED_LIBRARY,jsig), \
```

Comment:
This is redundant and should be removed: jsig is already in the `compressedrefs` (or `default`) subdirectory which is searched before the main `lib` directory.

Reply:
The call on line 183 puts it in the server (and as requested in the j9vm) directories; the same places users will find libjvm.so.